packaging/arch/PKGBUILD

```diff
@@ -195,4 +198,18 @@ package_snapd-git() {
   install -d -m 755 "$pkgdir/var/lib/snapd/seccomp/bpf"
   install -d -m 755 "$pkgdir/var/lib/snapd/snap/bin"
   install -d -m 755 "$pkgdir/var/lib/snapd/snaps"
+  install -d -m 755 "$pkgdir/var/lib/snapd/lib/gl"
+  # these dirs have special permissions
+  install -d -m 000 "$pkgdir/var/lib/snapd/void"
```

Comment:
❤️  about the permissions. I found that most packaging systems choke on this and need special magic to actually allow this mode.